src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Gateway/Pages/OrganisationCriminalComplianceChecks.cshtml

```diff
@@ -3,104 +3,36 @@
 @model SFA.DAS.AdminService.Web.ViewModels.Roatp.Gateway.OrganisationCriminalCompliancePageViewModel
 
 @{
-    var heading = "Organisation's criminal and compliance checks";
-    ViewBag.Title = $"Gateway - {heading}";
+    ViewBag.Title = $"Gateway - {Model.Heading}";
     Layout = "_Layout";
-
-    var formGroupError = Model.ErrorMessages != null && Model.ErrorMessages.Any() ? "govuk-form-group--error" : "";
-    var isErrorFailText = "";
-    var noSelectionMessage = "";
-    var failTextMessage = "";
-    if (Model.ErrorMessages != null && Model.ErrorMessages.Any())
-    {
-        foreach (var error in Model.ErrorMessages)
-        {
-            if (error.Field == "OptionPass")
-            {
-                noSelectionMessage = error.ErrorMessage;
-            }
-
-            if (error.Field == "OptionFailText")
-            {
-                isErrorFailText = "govuk-textarea--error-override";
-                failTextMessage = error.ErrorMessage;
-            }
-        }
-    }
 }
 
-<div class="govuk-breadcrumbs">
-    <ol class="govuk-breadcrumbs__list">
-        <li class="govuk-breadcrumbs__list-item">
-            <a class="govuk-breadcrumbs__link" href="/Dashboard">RoATP dashboard</a>
-        </li>
-        <li class="govuk-breadcrumbs__list-item">
-            <a class="govuk-breadcrumbs__link" href="/Roatp/Gateway/New">RoATP applications</a>
-        </li>
-        <li class="govuk-breadcrumbs__list-item">
-            <a class="govuk-breadcrumbs__link" href="/Roatp/Gateway/@Model.ApplicationId">Application assessment overview</a>
-        </li>
-        
-        <li class="govuk-breadcrumbs__list-item" aria-current="page">@Model.PageTitle</li>
-    </ol>
-</div>
+@{ await Html.RenderPartialAsync("~/Views/Roatp/Apply/Gateway/Pages/_GatewayAssessmentBreadcrumb.cshtml"); }
```

Comment:
Use the partial tag helper as Html.RenderPartialAsync is deprecated in ASP.NET Core